lib/models/addon.js

```diff
@@ -223,8 +223,11 @@ let addonProto = {
 
     let emberCLIBabelConfigKey = this._emberCLIBabelConfigKey();
     this.__originalOptions[emberCLIBabelConfigKey] = this.options[emberCLIBabelConfigKey] = defaultsDeep(this.options[emberCLIBabelConfigKey], {
-      compileModules: true,
+      compileModules: false,
+      disablePresetEnv: true,
```

Comment:
Can you also add `disableDebugTooling: true` to this list of options?

Reply:
done